tests/integration/tier0/backup-release/test_backup_release.py

```diff
@@ -24,6 +24,9 @@ def test_backup_os_release_no_envar(shell, convert2rhel):
     # Move all repos to other location so it is not used
     assert shell("mkdir /tmp/s_backup && mv /etc/yum.repos.d/* /tmp/s_backup/").returncode == 0
```

Comment:
kokesak, danmyway, and I talked about this today and we decided it was okay to stick with shell.  subprocess is the encouraged way to call out to other programs ( First paragraph of https://docs.python.org/3/library/subprocess.html ).  The other functions in the python stdlib are considered legacy ways of calling out.

The one thing about the `shell()` function to be aware of is that (as the name implies) it sends the command to the shell.  This means that shell metacharacters are processed by the shell (`*`, `;`, `&&`, etc).  This can be a source of security bugs whenever user input is used but I don't believe that's a problem for our integration tests in general.

Reply:
@abadger thanks, we will stick with the `shell()` function.